hacking/conf2yaml.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+
+import ast
+import yaml
+import os
+import sys
+from ansible.parsing.yaml.dumper import AnsibleDumper
+
+things = {}
+stuff = {}
+
+op_map = {
+    ast.Add: '+',
+    ast.Sub: '-',
+    ast.Mult: '*',
+    ast.Div: '/',
+}
+
+
+
```

Comment:
hehe, yes, thanks for that, i had my own but yours was better, so i just repurposed  it, no need to keep this after the merge, was just htere to allow me to 'migrate multiple times'